AdtSampleApp/SampleClientApp/Program.cs

```diff
@@ -68,28 +57,28 @@ static async Task Main()
                 {
                     client.GetDigitalTwin("---");
                 }
-                catch (RequestFailedException rex)
+                catch (RequestFailedException)
                 {
-
+                    // do nothing; this path is expected
```

Comment:
// As we are intentionally try to retrieve a twin that is most likely not going to exist, this exception is expected
// We just do this to force the authentication library to authenticate ahead